execution-frameworks/Invoke-AtomicRedTeam/Invoke-AtomicRedTeam.ps1

```diff
@@ -0,0 +1,162 @@
+<#
```

Comment:
Have you run something like [`PSScriptAnalyzer`](https://github.com/PowerShell/PSScriptAnalyzer) on this? I've found it tends to give fairly helpful tips.